tools/AutoTuner/installer.sh

```diff
@@ -0,0 +1,8 @@
+#!/bin/bash
+
```

Comment:
I think we need to add code here to consider that this can be called from ORFS root as well as AT's root.